src/EditorFeatures/CSharp/CSharpEditorResources.resx

```diff
@@ -437,4 +437,7 @@
   <data name="Outside_namespace" xml:space="preserve">
     <value>Outside namespace</value>
   </data>
+  <data name="Prefer_is_null_check_over_is_object" xml:space="preserve">
+    <value>Prefer 'is null' check over 'is object'</value>
```

Comment:
💭 Should we match semantics?

```suggestion
    <value>Prefer 'is not null' check over 'is object'</value>
```

Reply:
@sharwell I wasn't able to word it accurately in a small sentence. It's both ways (prefer `is null` over `is not object` and prefer `is not null` over `is object`).

Reply:
i would have both forms :)

Reply:
I don't love it. But I could live with it. @sharwell ?

My pref really would be a unique string for each, or something like "Prefer 'null' check over type check".

Reply:
i still like my suggestion here.

Reply:
I thought I already did that (using unique string for each), but turns out this resource file is for editorconfig UI, and what I did was for the analyzer. Actually, a unique string cannot be done in EditorConfig UI.

I'll use "Prefer 'null' check over type check" for both the analyzer and EditorConfig UI.